common/src/main/java/com/wynntils/features/overlays/ShamanTotemTrackingFeature.java

```diff
@@ -78,29 +69,22 @@ public void onTotemSummoned(TotemEvent.Summoned e) {
         totemAS.setSharedFlag(ENTITY_GLOWING_FLAG, true);
     }
 
-    @SubscribeEvent
-    public void onTick(TickEvent event) {
-        shamanTotemTimerOverlay.ticksUntilUpdate--;
-
-        if (shamanTotemTimerOverlay.ticksUntilUpdate <= 0) {
-            shamanTotemTimerOverlay.ticksUntilUpdate = ShamanTotemTimerOverlay.TICKS_PER_UPDATE;
-            shamanTotemTimerOverlay.updateRenderTaskCache();
-        }
-    }
-
-    public static class ShamanTotemTimerOverlay extends Overlay {
-        static final int TICKS_PER_UPDATE = 5;
-
+    public static class ShamanTotemTimerOverlay extends TextOverlay {
         @Config
         public static TotemTrackingDetail totemTrackingDetail = TotemTrackingDetail.COORDS;
 
         @Config
-        public TextShadow textShadow = TextShadow.OUTLINE;
+        public static ChatFormatting firstTotemTextColor = ChatFormatting.WHITE;
 
-        private TextRenderSetting textRenderSetting;
+        @Config
+        public static ChatFormatting secondTotemTextColor = ChatFormatting.BLUE;
+
+        @Config
+        public static ChatFormatting thirdTotemTextColor = ChatFormatting.RED;
```

Comment:
This is the only "regression" we have. This feature uses CustomColor for totem colors, but having multiple CustomColors per TextOverlay is not possible at the moment. I toyed with a few ways to implement it, but I did not like either of the implementations. So, for the time being, this was added. Hopefully, we can remove it sooner or later.